examples/todo/src/__tests__/acceptance/todo-with-id.acceptance.ts

```diff
@@ -0,0 +1,198 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/example-todo
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {EntityNotFoundError} from '@loopback/repository';
+import {
+  Client,
+  createRestAppClient,
+  expect,
+  givenHttpServerConfig,
+  toJSON,
+} from '@loopback/testlab';
+import {TodoListApplication} from '../../application';
+import {TodoWithId} from '../../models';
+import {TodoWithIdRepository} from '../../repositories';
+import {
+  getProxiedGeoCoderConfig,
+  givenCachingProxy,
+  givenTodoWithId,
+  HttpCachingProxy,
+} from '../helpers';
+
+describe('TodoApplication - test TodoWithId', () => {
```

Comment:
How does `TodoWithId` fit into the narrative of our [Todo tutorial](https://loopback.io/doc/en/lb4/todo-tutorial.html)?

In my mind, the primary purpose of Todo example app is to support the tutorial and show a simple & meaningful LB4 application. It's not the best place for adding acceptance tests that are not related to the Todo tutorial.